vsts/scripts/notifyBuildFailure.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+# --------------------------------------------------------------------------------------------
+
+set -e
+
+declare -r createBugFileContent='[{"op": "add","path": "/fields/System.Title","from": null,"value": "Build Failed-'$BUILD_DEFINITIONNAME'-'$BUILD_BUILDNUMBER'"}, \
+                                                                  {"op": "add", path": "/fields/System.AreaPath", "value": "DevDiv\Azure Developer Experience\Oryx"}, \
+                                                                  {"op": "add", "path": "/fields/Microsoft.DevDiv.OrgRank","value": "9999"}]'
+if [ "$BUILD_SOURCEBRANCH" == "refs/heads/master" ] && [ "$BUILD_REASON" == "Schedule" ] && \
+   ( [ "$BUILD_DEFINITIONNAME" == "Oryx-Nightly" ] || [ "$BUILD_DEFINITIONNAME" == "Oryx-CI" ] ); then
+        echo "here i am"
+        token=$DEVDIV_TOKEN
```

Comment:
Is this stored as a secret in key vault?

Reply:
in the build agent env variable